src/VisualStudio/IntegrationTest/TestUtilities/ScreenshotService.cs

```diff
@@ -28,12 +33,23 @@ public static void TakeScreenshot(string fullPath)
         /// <summary>
         /// Captures the full screen to a <see cref="Bitmap"/>.
         /// </summary>
-        /// <returns>A <see cref="Bitmap"/> containing the screen capture of the desktop.</returns>
-        private static Bitmap CaptureFullScreen()
+        /// <returns>
+        /// A <see cref="Bitmap"/> containing the screen capture of the desktop, or null if a screen
+        /// capture can't be created.
+        /// </returns>
+        private static Bitmap TryCaptureFullScreen()
         {
-            var bitmap = new Bitmap(
-                width: Screen.PrimaryScreen.Bounds.Width,
-                height: Screen.PrimaryScreen.Bounds.Height);
+            int width = Screen.PrimaryScreen.Bounds.Width;
+            int height = Screen.PrimaryScreen.Bounds.Height;
+
+            if (width < 0 || height < 0)
```

Comment:
Blah. <= is what I meant; clearly my fingers didn't get the message while I was typing.